modules/ROOT/pages/tgw-manage-arm.adoc

```diff
@@ -0,0 +1,113 @@
+= Manage Transit Gateway Attachments
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+image:logo-cloud-active.png[xref="deployment-strategies.adoc#cloudhub-deployments", title="CloudHub"]
+image:logo-hybrid-disabled.png[xref="deployment-strategies.adoc#hybrid-deployments", title="Hybrid"]
+image:logo-server-disabled.png[xref="deployment-strategies.adoc#anypoint-platform-pce-deployments", title="Anypoint Platform PCE"]
+image:logo-rtf-disabled.png[xref="deployment-strategies.adoc#anypoint-runtime-fabric-deployments", title="Runtime Fabric"]
+
+
+Use Anypoint Runtime Manager to manage transit gateway attachments.
+
+== Rename a Transit Gateway Attachment
+
+You might want to rename a transit gateway attachment to match the attachment name in AWS. 
+
+To rename a transit gateway attachment:
+
+. On the *Transit Gateways* page, click the Transit Gateway menu (*...*) and select *Edit name*:
++
+image::tgw-rename.png[Edit name option in the Transit Gateways menu]
+. In the popup window, enter a new name and click *Save*.
+
+== Add a Route to the Transit Gateway Route Table
+
+. In Anypoint Platform, on the *Transit Gateways* page, in the *VPCs* section, click *Add Route*. 
+. On the *Add VPC Route* window, use CIDR notation to enter a single subnet for your
+AWS VPC:
++
+image::tgw-add-vpc-route.png[Add VPC route]
+. Click *Add Route*.
++
+Anypoint Platform submits the new route to AWS.
+
+When the route addition succeeds, the *Transit Gateways* page shows the *Route added* message
+and the *Transit Gateways* page shows the route CIDR in the route table.
+
+In the route table, added routes show *Transit Gateway* as the value in the *Next Hop* column:
+
+image::tgw-route-added.png[Transit Gateway in the Next Hop column of the route table]
+
+== Remove a Route from the Transit Gateway Route Table
+
+You might need to remove a route if you no longer want a subnet to be accessible through the transit gateway.
+
+To remove a route:
+
+. On the *Transit Gateways* page, in the *VPCs* section, click the *X* icon in the appropriate Transit Gateway route row of the route table:
++
+image::tgw-remove-vpc-route.png[Remove route icon in the VPCs section]
+. At the confirmation prompt, click *Remove*.
+
+== Detach an Anypoint VPC from the Transit Gateway
+
+You might need to detach a VPC so that you can reattach it.
+
+Detaching a VPC from a transit gateway also deletes the attachment in AWS.
+Any apps deployed to the VPC continue to run, but they are no longer connected to your Amazon VPCs and VPNs.
+
+To detach a VPC from the transit gateway:
```

Comment:
Copying this to use an example of well-crafted "delete" task workup, which I often see presented as a simple "do this" without any reasoning...or with so much reasoning that the actual task gets lost....